Makefile

```diff
@@ -16,15 +16,23 @@ DOCKER := $(shell command -v docker)
 	echo "DISPLAY=$(DISPLAY)" >> .env
 	echo "CASA_VERSION=$(CASA_VERSION)" >> .env
 
-compose: .env xhost ensureborders down  # Is this "down" really necessary? .env runs clean, which runs stop, running "docker compose down || true"
-
+# first function without dot is assigned to command "make"=="make compose".
+# This function builds the container and applies changes to other files.
+compose: .env xhost ensureborders
 	docker compose build
 	docker compose up -d casa
```

Comment:
instead of requiring `ensureborders`, `make` can call `make start` here or equivalent.

so we have those compose cmds in a single place